Formula/arangodb.rb

```diff
@@ -23,41 +26,91 @@ class Arangodb < Formula
   end
 
   def install
-    # clang on 10.8 will still try to build against libstdc++,
-    # which fails because it doesn't have the C++0x features
-    # arangodb requires.
     ENV.libcxx
 
-    args = %W[
-      --disable-dependency-tracking
-      --prefix=#{prefix}
-      --disable-relative
-      --datadir=#{share}
-      --localstatedir=#{var}
-    ]
+    if build.head?
+      args = %W[
```

Comment:
I'm not _enormously_ in love with a significant list of custom `cmake` args, to be honest. We try to nudge people towards `std_cmake_args` because it provides us with consistency.